framework/test/test_verify_bugs.sh

```diff
@@ -16,8 +16,11 @@
 #   * Verify bug 2 with DEBUG  ./test_verify_bugs.sh -pLang -b 2 -D
 #
 ################################################################################
+
+HERE=$(cd `dirname $0` && pwd)
+
 # Import helper subroutines and variables, and init Defects4J
-source test.include
+source "$HERE/test.include" || exit 1
 
```

Comment:
Should there be a call to `init` here as well?

Reply:
Good point, I missed this earlier. Unless there is a reason for why init should happen after argument processing, I think we should move this up so that source and init pattern is recognizable and the same for all scripts.